test/Main.purs

```diff
@@ -36,11 +36,11 @@ main = do
     altTest2 =
       Right unit <|> unsafeThrow "Too strict!"
 
-  when false \do
+  when false \_ ->
     unsafeThrow "Too strict!"
 
-  unless true \do
+  unless true \_ ->
     unsafeThrow "Too strict!"
 
-  guard false \do
+  guard false \_ ->
```

Comment:
Did you intend to add `\\` as an operator? The point of these tests is to test that the operator works, so by not using the operator, they no longer test anything ;).

Reply:
Fixed, I though these were testing guard/when/unless